### ISSUE TITLE
[TalkBack] Adjusting a seek control using double-tap does not provide spoken feedback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#185]
```
What steps will reproduce the problem?
1. while using talk back, locate a seek control with a percentage value in 
android.
2. Tap any ware on the seek control. You can also double tap and hold the 
control while dragging your finger along it.

What is the expected output?
Talkback should be able to either have some way to toggle feedback settings or 
just announce the percentage value you have moved to so you don't have to 
scroll (Flick or drag) away from the seek control and back again.. What do you 
see instead?
Talk only gives the user audible feedback to tell you that the seek control's 
and/or the slider value is being adjusted.

What version of the product are you using? On what operating system?
talkback-v3.3.0_b2_r87
Android 4.2.2 stock firmware.
Please provide any additional information below.

```

Original issue reported on code.google.com by `jde2...@gmail.com` on 24 Feb 2013 at 7:34

**Comment by GoogleCodeExporter** [comment]
```
TalkBack should provide feedback as a high-pitched "X percent" about half a 
second after you stop adjusting a seek control. Where are you encountering this 
bug?
```

Original comment by `alanv@google.com` on 6 Mar 2013 at 1:30

**Comment by GoogleCodeExporter** [comment]
```
I encounter it in all seek controls. Talk back will give audible feedback,
but it would be use full if it would read the percentage also so you don't
have to move away and back again to find out the percentage with speech.
```

Original comment by `jde2...@gmail.com` on 6 Mar 2013 at 2:12

**Comment by GoogleCodeExporter** [comment]
```
Was able to reproduce by touching the desired seek location and then 
double-tapping anywhere on the screen. This may be related to silencing speech 
on first touch, will research further.

If you use the "next page" gesture (swipe right and then left in a single 
gesture) while you have a seek control focused, do you hear any feedback?
```

Original comment by `alanv@google.com` on 8 Mar 2013 at 7:24
- Changed title: **[TalkBack] Adjusting a seek control using double-tap does not provide spoken feedback**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I'll have to try it and let you know. I'd never thought of trying that.
```

Original comment by `jde2...@gmail.com` on 8 Mar 2013 at 4:28

**Comment by GoogleCodeExporter** [comment]
```
Yes, I do get spoken feedback wen using the next page gesture, but I find it 
slightly hard to use and not because I can't perform it, but because my Nexus 
doesn't always seem to take it.
```

Original comment by `jde2...@gmail.com` on 8 Mar 2013 at 5:43

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 30 Apr 2013 at 8:09

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 29 May 2013 at 8:07
